temporal-opentracing/src/main/java/io/temporal/opentracing/internal/OpenTracingWorkflowOutboundCallsInterceptor.java

```diff
@@ -48,7 +49,7 @@ public OpenTracingWorkflowOutboundCallsInterceptor(
 
   @Override
   public <R> ActivityOutput<R> executeActivity(ActivityInput<R> input) {
-    if (!Workflow.isReplaying()) {
+    if (!WorkflowUnsafe.isReplaying()) {
```

Comment:
Should we expose this as `Workflow.unsafe.isReplaying()`?

Reply:
I think the class level is common in Java.

Reply:
Do you mean like `Workflow.Unsafe.isReplaying` where `Unsafe` is an inner class of `Workflow`?
We can, but it will actually increase the discoverability (people will see it autocomplete for `Workflow.`), so I'm in favor of a separate class.